### PR TITLE
Fix #125, 123, 112, 127, 131

### PR DESCRIPTION
**Pull request by aydevelop**

### CODE REVIEW

File: backend/Recodo.BLL/Services/EmailService.cs [hunk 1]
```diff
@@ -9,15 +10,23 @@ public class EmailService
...
+                var to = new EmailAddress(toEmail, toEmail);
 
-            var msg = MailHelper.CreateSingleEmail(from, to, subject, message, message);
-            await client.SendEmailAsync(msg);
+                var msg = MailHelper.CreateSingleEmail(from, to, subject, message, message);
+                await client.SendEmailAsync(msg);
+            }
+            catch (System.Exception)
+            {
+                //For debugging
```
Comment (Anatolii-Ivanov) [inline backend/Recodo.BLL/Services/EmailService.cs]: Remove please

File: frontend/src/app/modules/auth/login-page/login-page.component.html [hunk 2]
```diff
@@ -43,7 +43,9 @@ <h2>Sign in to Recodo</h2>
 
         <p class="input-label">
           Password
-          <a routerLink="/reset-password" href="#" class="forgot-password">(Forgot password)</a>
+          <a routerLink="/reset-password" href="#" class="forgot-password"
+            >(Forgot password)</a
+          >
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/auth/login-page/login-page.component.html]:
```
<a routerLink="/reset-password" href="#" class="forgot-password">
            (Forgot password)
</a>
```

File: frontend/src/environments/environment.ts [hunk 3]
```diff
@@ -3,8 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  apiUrl: 'https://localhost:44316/api',
-  blobApiUrl: 'https://localhost:44362/api',
+  apiUrl: 'https://localhost:5001/api',
+  blobApiUrl: 'https://localhost:5001/api',
```
Comment (Anatolii-Ivanov) [inline frontend/src/environments/environment.ts]: I think it was used for development, please don't change it globally for now

**Comment by aydevelop** [comment]
Untitled
Here is fix

**Comment by aydevelop** [comment]
FIxed!

File: frontend/src/app/modules/workspace/settings/settings.component.ts [hunk 4]
```diff
@@ -232,17 +232,19 @@ export class SettingsComponent implements OnInit {
...
-        this.loginService.logOut();
-        this.snackbarService.openSnackBar('User deleted successfully');
-      },
-      error: () => {
-        this.snackbarService.openSnackBar(
-          'User not deleted. Something went wrong'
-        );
-      },
-    });
+    if (confirm('Are you sure you want to delete the user ?')) {
```
Comment (Jyuart) [inline frontend/src/app/modules/workspace/settings/settings.component.ts]: Almost 100% sure our PO won't like it this way.
Please use MatDialog

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by Jyuart — APPROVED** [review]

**Pull request merged**